js/apps/system/_admin/aardvark/APP/react/yarn.lock

```diff
@@ -3366,9 +3390,9 @@
   integrity sha512-XYmBiy+ohOR4Lh5jE379fV2IU+6Jn4g5qASinhitfyO71b/sCo6MKsMLF5tc7Zf2CE8hViVQyYSobJNke8OvUw==
 
 "@types/node@>=13.13.4":
-  version "16.4.5"
-  resolved "https://registry.npmjs.org/@types/node/-/node-16.4.5.tgz"
-  integrity sha512-+0GPv/hIFNoy8r5MFf7vRpBjnqNYNrlHdetoy23E7TYc7JB2ctwyi3GMKpviozaHQ/Sy2kBNUTvG9ywN66zV1g==
+  version "18.15.10"
```

Comment:
@pluma this should not have any effect right? This is post merge, yarn resolved it on it's own